### PR TITLE
Adds Pretty Filters for IC chat methods

### PR DESCRIPTION
**Pull request by utahclock**
:cl:
tweak: tweaks pretty filter to include IC text.
/:cl:

### CODE REVIEW

**Comment by Super3222** [comment]
admin abuse

**Comment by Kmc2000** [comment]
For those that dont know, pretty filters stop you using bigotry

**Comment by Jc1469** [comment]
It'd probably be better to just leave IC as it is.  Replacing IC text makes an IC mess if it's set for bigotry.
Other than that, it's just another niche meme admin tool for trolling at the end of a round.

**Comment by oisin100** [comment]
If we do this we should alert admins when someone trips a pretty filter.

**Comment by matskuman5** [comment]
"If we do this we should alert admins when someone trips a pretty filter."

^^^^^^^^

Otherwise logs are fucked and people can just say "oh it was a joke i just typed in BANMEADMINS and hit enter"

**Comment by utahclock** [comment]
I understand this is controversial. I suppose I can an admin message trigger. Not sure how to get it to state x said n word which triggered filter. The n word part would be part i would need help with . if any one has ideas.. On how to do that pm me on discord.

File: code/modules/mob/say.dm [hunk 1]
```diff
@@ -2,19 +2,31 @@
...
+	log_say("[name] : [message]")
 	if(say_disabled)	//This is here to try to identify lag problems
 		usr << "<span class='danger'>Speech is currently admin-disabled.</span>"
 		return
 	usr.say(message)
 
+
+
+
+
```
Comment (monster860) [inline code/modules/mob/say.dm]: **screams**

AAAAAAAAAAAA! Too many blank lines!!!

File: code/modules/mob/say.dm [hunk 2]
```diff
@@ -2,19 +2,31 @@
...
+	log_whisper("[name] : [message]")
 	if(say_disabled)	//This is here to try to identify lag problems
 		usr << "<span class='danger'>Speech is currently admin-disabled.</span>"
 		return
 	say(message) //only carbons actually whisper, everything else just talks
 
+
+
+
+
```
Comment (monster860) [inline code/modules/mob/say.dm]: **screams**

File: code/modules/mob/say.dm [hunk 3]
```diff
@@ -27,9 +39,12 @@
 
 	usr.emote("me",1,message)
 
+
```
Comment (monster860) [inline code/modules/mob/say.dm]: **screams**

**Review by monster860 — CHANGES_REQUESTED** [review]
You're adding blank lines for 4 no raisin.

**Comment by Super3222** [comment]
tbh I don't like it.

**Comment by Time-Green** [comment]
I see seriously no problem with this?
Do you guys want people to scream F----- out loud? Isn't that against the rules?
Just notify admins if they do.

**Comment by Super3222** [comment]
IC chatter shouldn't be hindered. One of the many things to laugh about is someone using that word, and than getting banned for it. Filtering it so unfun, and while it may be against the rules we aren't Nazis, and nobody is going to get triggered over it. I am all for the dedication against slurs and other degrading profanities but if they do get triggered than they should stay off SS13.

At the very least an admin notification should be sent to ASAY. Nothing more

**Pull request merged**